code/game/turfs/snow.dm

```diff
@@ -168,19 +163,22 @@
 
 	switch(burnlevel)
 		if(1 to 10)
-			slayer--
+			slayer = max(0, slayer - 1)
+			update_icon(1, 0)
 		if(11 to 24)
 			slayer = max(0, slayer - 2)
+			update_icon(1, 0)
 		if(25 to INFINITY)
 			slayer = 0
+			update_icon(1, 0)
 
 /turf/open/floor/plating/ground/snow/proc/acidspray_act()
-	SIGNAL_HANDLER
```

Comment:
why remove that signal handler ?

Reply:
It works, but it's ultimately not necessary and linters yells at me